Description: Disable install and update options
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2022-11-06

--- commix-3.5.orig/src/core/main.py
+++ commix-3.5/src/core/main.py
@@ -27,10 +27,10 @@ from src.thirdparty.six.moves import url
 from src.utils import menu
 from src.utils import logs
 from src.utils import purge
-from src.utils import update
+#from src.utils import update
 from src.utils import common
 from src.utils import version
-from src.utils import install
+#from src.utils import install
 from src.utils import crawler
 from src.utils import settings
 from src.core.requests import parameters
@@ -635,13 +635,13 @@ try:
       raise SystemExit()
       
     # Check if defined "--update" option.        
-    if menu.options.update:
-      update.updater()
+#    if menu.options.update:
+#      update.updater()
         
     # Check if defined "--install" option.        
-    if menu.options.install:
-      install.installer()
-      raise SystemExit()
+#    if menu.options.install:
+#      install.installer()
+#      raise SystemExit()
 
     if not sys.stdin.isatty():
       settings.IS_TTY = False
@@ -654,7 +654,7 @@ try:
     if settings.IS_TTY and not any((menu.options.url, menu.options.logfile, menu.options.bulkfile, \
                 menu.options.requestfile, menu.options.sitemap_url, menu.options.wizard, \
                 menu.options.update, menu.options.list_tampers, menu.options.purge, menu.options.noncore_dependencies)):
-      err_msg = "Missing a mandatory option (-u, -l, -m, -r, -x, --wizard, --update, --list-tampers, --purge or --dependencies). "
+      err_msg = "Missing a mandatory option (-u, -l, -m, -r, -x, --wizard, --list-tampers, --purge or --dependencies). "
       err_msg += "Use -h for help."
       print(settings.print_critical_msg(err_msg))
       raise SystemExit()
@@ -816,8 +816,8 @@ try:
       parser.logfile_parser()
 
     # Check if ".git" exists and check for updated version!
-    if os.path.isdir("./.git") and settings.CHECK_FOR_UPDATES_ON_START:
-      update.check_for_update()
+    #if os.path.isdir("./.git") and settings.CHECK_FOR_UPDATES_ON_START:
+    #  update.check_for_update()
 
     # Check if option is "--url" for single url test.
     if menu.options.sitemap_url:
@@ -949,4 +949,4 @@ except EOFError:
   print(settings.print_error_msg(err_msg))
   raise SystemExit()
 
-# eof
\ No newline at end of file
+# eof
--- commix-3.5.orig/src/core/shells/reverse_tcp.py
+++ commix-3.5/src/core/shells/reverse_tcp.py
@@ -545,7 +545,7 @@ commix(""" + Style.BRIGHT + Fore.RED + "
                       break 
               except:
                 unicorn_version = "" 
-              update.check_unicorn_version(unicorn_version)
+              #update.check_unicorn_version(unicorn_version)
               try:
                 if len(unicorn_version) == 0:
                   unicorn_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../', 'thirdparty/unicorn'))
--- commix-3.5.orig/src/utils/menu.py
+++ commix-3.5/src/utils/menu.py
@@ -60,21 +60,21 @@ general.add_option("-v",
                 dest="verbose",
                 help="Verbosity level (0-4, Default: 0).")
 
-general.add_option("--install",
-                action="store_true",
-                dest="install",
-                default=False,
-                help="Install " + settings.APPLICATION + " to your system.")
+#general.add_option("--install",
+#                action="store_true",
+#                dest="install",
+#                default=False,
+#                help="Install " + settings.APPLICATION + " to your system.")
 
 general.add_option("--version",
                 action="store_true",
                 dest="version",
                 help="Show version number and exit.")
 
-general.add_option("--update", 
-                action="store_true",
-                dest="update",
-                help="Check for updates (apply if any) and exit.")
+#general.add_option("--update", 
+#                action="store_true",
+#                dest="update",
+#                help="Check for updates (apply if any) and exit.")
 
 general.add_option("--output-dir", 
                 action="store",
