From: Kali Developers <devel@kali.org>
Date: Tue, 12 Nov 2019 15:41:43 +0100
Subject: fix-usage-example

---
 src/utils/menu.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: commix/src/utils/menu.py
===================================================================
--- commix.orig/src/utils/menu.py
+++ commix/src/utils/menu.py
@@ -45,7 +45,7 @@ Copyright © """ + settings.YEAR + """ "
 
 _ = os.path.normpath(sys.argv[0])
 
-usage = "python %prog [option(s)]"
+usage = "commix [option(s)]"
 
 parser = OptionParser(usage=usage)
 
